networking/hardware_networks/configuring-sriov-device.adoc

```diff
@@ -0,0 +1,16 @@
+[id="configuring-sriov-device"]
+= Configuring an SR-IOV network device
+include::modules/common-attributes.adoc[]
+:context: about-sriov
+
+toc::[]
+
+You can configure a Single Root I/O Virtualization (SR-IOV) device in your cluster.
+
+include::modules/nw-sriov-configuring-device.adoc[leveloffset=+1]
+
+include::modules/nw-sriov-add-pod-runtimeconfig.adoc[leveloffset=+1]
```

Comment:
Please move `include::modules/nw-sriov-add-pod-runtimeconfig.adoc` to `networking/hardware_networks/configuring-sriov-net-attach.adoc` file and put it after `include::modules/nw-multus-ipam-object.adoc`.

The content in `nw-sriov-add-pod-runtimeconfig` is mainly about how to configure sriov net-attach-def with static MAC and IP addresses.